src/plugin.js

```diff
@@ -45,7 +46,13 @@ class WebpackRequireFrom {
   }
 
   compilationHook(compilation) {
-    const { mainTemplate } = compilation;
+    const { mainTemplate, name } = compilation;
+
+    if(this.options[DISABLE_CSS_OPTION_NAME]) {
+      // skip further execution for CSS compilation
+      if(/\.(sa|sc|c)ss$/i.test(name))
+        return;
```

Comment:
`mini-css-extract-plugin` and `extract-css-chunks-webpack-plugin` compilation sets name as 
 `<plugin_name> <css_file>`. This is done for each CSS file referenced via JS *import* of *require* statement.
So we could determine if CSS files are being compiled from the name field,